.circleci/config.yml

```diff
@@ -347,13 +351,13 @@ workflows:
               only: master
       - license-check
       - ui
-      - runtime
       - core:
+      - runtime
           requires:
-            - runtime
+            - core
```

Comment:
don't think that runtime requires core now (it was just @chirino's PR, but I hope that we can avoid tha)